test/langtools/tools/javac/Paths/ClassPath.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (c) 2003, 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+
+/*
+ * @test
+ * @bug 4212732 8293877
+ * @summary Test handling of the Class-Path attribute in jar file manifests
+ * @library /tools/lib
+ * @build toolbox.ToolBox Util ClassPath
+ * @run main ClassPath
+*/
+
+
+/*
+ * Converted from Class-Path.sh, originally written by Martin Buchholz.
+ *
+ * For the last version of the original, Class-Path.sh, see
+ * https://github.com/openjdk/jdk/blob/jdk-19%2B36/test/langtools/tools/javac/Paths/Class-Path.sh
```

Comment:
Please change this and other github URLs to use "https://git.openjdk.org/jdk" as the prefix, e.g.:

https://git.openjdk.org/jdk/blob/jdk-19%2B36/test/langtools/tools/javac/Paths/Class-Path.sh

Reply:
Darn!  missed this; will have to fix separately